conf/application.conf

```diff
@@ -129,13 +129,18 @@ mail.reply = "No reply <no-reply@webknossos.org>"
 play.http.parser.maxMemoryBuffer = 250M
 play.http.parser.maxDiskBuffer = 1G
 
-postgres = {
-  url = "jdbc:postgresql://localhost/webknossos"
-  driver = org.postgresql.Driver
-  keepAliveConnection = true
-  user = "postgres"
-  password = "postgres"
-  queueSize = 5000
+slick = {
+  profile = "slick.jdbc.PostgresProfile$"
+  codegen.package = "com.scalableminds.webknossos.schema"
+  db = {
+    url = "jdbc:postgresql://localhost/webknossos"
+    url = ${?POSTGRES_URL}
```

Comment:
Yes, cf [this stackoverflow answer](https://stackoverflow.com/questions/38197406/overriding-configuration-with-environment-variables-in-typesafe-config). And yes, this might mean that the line in docker-compose is obsolete now. I will test that :)

Reply:
Yes, seems to work too